api/api.py

```diff
@@ -930,6 +944,13 @@ def custom_build(ref: str, command: str, deps: List[str], tag: str = "", disable
       and inject that image ref into the YAML. For more on content-based tags, see <custom_build.html#why-tilt-uses-immutable-tags>_
     command_bat: The command to run, expressed as a Windows batch command executed
       with ``cmd /S /C``. Takes precedence over the ``command`` parameter on Windows. Ignored on macOS/Linux.
+    image_deps: a list of image builds that this deploy depends on.
+      The tagged image names will be injected into the environment of the
+      the custom build command in the form:
+
+      `TILT_IMAGE_i` - The reference to the image #i from the point of view of the local host.
```

Comment:
```suggestion
      `TILT_IMAGE_i` - The reference to the image #i (0-based) from the point of view of the local host.
```

"#i" would lead me to try 1-based before 0-based